tests/Service/Parser/FunctionParserTest.php

```diff
@@ -157,12 +170,35 @@ public function shouldNotSupportWithoutQueryParameters(): void
      */
     public function shouldNotParseWithoutQueryParameters(): void
     {
-        $this->markTestSkipped('Skipped because there are no parsers yet');
         $this->expectException(InvalidArgumentException::class);
         $this->expectExceptionMessage('Query parameters must be set to FunctionParser.');
 
         foreach ($this->parser->parse('foo', 'bar') as $filter) {
             $this->fail('This should not be reached');
         }
     }
+
+    /**
+     * @test
+     */
+    public function shouldNotParseFunctionByExplicitValueDefinition(): void
+    {
+        // ?fun=fullName&firstName=Jon&surname=Snow
+        $queryParameters = ['fun' => 'fullName', 'firstName' => 'Jon', 'surname' => 'Snow'];
```

Comment:
It will be longer to write and since it is a query parameter column, it metters (_query parameters has max length of 2048 chars_). There are more places for typo and I generally see it more fun to use `fun` instead of `function`..

But I dont really care what it is .. so ..  ¯\\_(ツ)_/¯